src/Turbo/Mercure/TurboStreamListenRenderer.php

```diff
@@ -28,31 +26,25 @@ final class TurboStreamListenRenderer implements TurboStreamListenRendererInterf
 {
     private $hub;
     private $stimulusTwigExtension;
-    private $propertyAccessor;
-    private $doctrine;
+    private $twigBroadcaster;
 
-    public function __construct(HubInterface $hub, StimulusTwigExtension $stimulusTwigExtension, PropertyAccessorInterface $propertyAccessor = null, ManagerRegistry $doctrine = null)
+    public function __construct(HubInterface $hub, StimulusTwigExtension $stimulusTwigExtension, TwigBroadcaster $twigBroadcaster)
```

Comment:
This should be moved to an independent service dedicated to extract the ID, but I can do it in a follow-up PR.

Reply:
I created a new class, `IdAccessor` for thi purpose. it's not final to allow extensibility via inheritance. I did not add an interface because it would look overkill to me, at this stage at least.